### PR TITLE
LOOP-26 Upload Loop dosing decision data to Tidepool backend

### PR DESCRIPTION
**Pull request by darinkrauss**
- https://tidepool.atlassian.net/browse/LOOP-26
- Minor updates due to LoopKit StoredDosingDecision changes

### CODE REVIEW

File: NightscoutServiceKit/Extensions/StoredDosingDecision.swift [hunk 1]
```diff
@@ -35,12 +35,19 @@ extension StoredDosingDecision {
     }
     
     var recommendTempBasal: RecommendedTempBasal? {
-        guard let tempBasalRecommendationDate = tempBasalRecommendationDate else {
+        guard let recommendedTempBasal = recommendedTempBasal else {
```
Comment (rickpasetto) [inline NightscoutServiceKit/Extensions/StoredDosingDecision.swift:38]: [nit] Ooh, this was hard to see that it wasn't an infinite loop; that _this_ variable is named `recommendTempBasal`, but it is calling `recommendedTempBasal`.  The old names were actually a little clearer to me.

File: NightscoutServiceKit/Extensions/StoredDosingDecision.swift [hunk 2]
```diff
@@ -35,12 +35,19 @@ extension StoredDosingDecision {
...
         }
-        return RecommendedTempBasal(timestamp: tempBasalRecommendationDate.date,
-                                    rate: tempBasalRecommendationDate.recommendation.unitsPerHour,
-                                    duration: tempBasalRecommendationDate.recommendation.duration)
+        return RecommendedTempBasal(timestamp: recommendedTempBasal.date,
+                                    rate: recommendedTempBasal.recommendation.unitsPerHour,
+                                    duration: recommendedTempBasal.recommendation.duration)
+    }
+
+    var recommendBolus: Double? {
```
Comment (rickpasetto) [inline NightscoutServiceKit/Extensions/StoredDosingDecision.swift]: [nit] Similarly here: `recommendBolus` and `recommendedBolus` are really hard to tell apart.

**Review by rickpasetto — APPROVED** [review]
Two nit suggestions.  I don't have much context here so I don't know what better names might be, but the naming is a bit confusing to me.  But I wouldn't block checking it in.

File: NightscoutServiceKit/Extensions/StoredDosingDecision.swift (hunk 1 above)
Comment (darinkrauss) [reply NightscoutServiceKit/Extensions/StoredDosingDecision.swift:38]: I renamed everything so it is clear.

File: NightscoutServiceKit/Extensions/StoredDosingDecision.swift (hunk 2 above)
Comment (darinkrauss) [reply NightscoutServiceKit/Extensions/StoredDosingDecision.swift]: I renamed everything so it is clear.

**Comment by darinkrauss** [comment]
@rickpasetto I renamed everything so it is clear. Would you please review those changes?

**Review by nhamming — APPROVED** [review]
LGTM

**Review by rickpasetto — APPROVED** [review]
lgtm

**Pull request merged**